### PR TITLE
CORE-5343: Minor changes to `KeysRpcOps`

### PR DESCRIPTION
**Pull request by vkolomeyko**

### CODE REVIEW

File: components/membership/membership-http-rpc/src/main/kotlin/net/corda/membership/httprpc/v1/KeysRpcOps.kt [hunk 1]
```diff
@@ -161,10 +164,11 @@ interface KeysRpcOps : RpcOps {
      */
     @HttpRpcGET(
         path = "{tenantId}/{keyId}",
-        description = "GET key in PEM format."
+        description = "GET key in PEM format.",
+        responseDescription = "The public key in PEM format."
```
Comment (alexey-kadyrov-r3) [inline components/membership/membership-http-rpc/src/main/kotlin/net/corda/membership/httprpc/v1/KeysRpcOps.kt:168]: Do we need to describe what response would be if the key is not found?

File: components/membership/membership-http-rpc/src/main/kotlin/net/corda/membership/httprpc/v1/KeysRpcOps.kt [hunk 2]
```diff
@@ -24,17 +24,18 @@ interface KeysRpcOps : RpcOps {
      */
     @HttpRpcGET(
         path = "{tenantId}/schemes/{hsmCategory}",
-        description = "Get list of schemes for the cluster."
+        description = "Get list of schemes for the cluster.",
+        responseDescription = "List of scheme names."
```
Comment (alexey-kadyrov-r3) [inline components/membership/membership-http-rpc/src/main/kotlin/net/corda/membership/httprpc/v1/KeysRpcOps.kt]: @charlieR3 what this operation does?

Comment (charlieR3) [reply components/membership/membership-http-rpc/src/main/kotlin/net/corda/membership/httprpc/v1/KeysRpcOps.kt]: These all call to the crypto ops client. This one does:
```
override fun listSchemes(
        tenantId: String,
        hsmCategory: String,
    ): Collection<String> = cryptoOpsClient.getSupportedSchemes(
        tenantId = tenantId,
        category = hsmCategory.uppercase()
    )
```

Comment (charlieR3) [reply components/membership/membership-http-rpc/src/main/kotlin/net/corda/membership/httprpc/v1/KeysRpcOps.kt]: Crypto ops client: `Returns the list of schemes codes which are supported by the associated HSM integration.`

Comment (alexey-kadyrov-r3) [reply components/membership/membership-http-rpc/src/main/kotlin/net/corda/membership/httprpc/v1/KeysRpcOps.kt]: Then the description and the responseDescription should match the crypto OPS

File: components/membership/membership-http-rpc/src/main/kotlin/net/corda/membership/httprpc/v1/KeysRpcOps.kt (hunk 1 above)
Comment (vkolomeyko) [reply components/membership/membership-http-rpc/src/main/kotlin/net/corda/membership/httprpc/v1/KeysRpcOps.kt:168]: We can put as much or as little description as necessary.

Comment (alexey-kadyrov-r3) [reply components/membership/membership-http-rpc/src/main/kotlin/net/corda/membership/httprpc/v1/KeysRpcOps.kt:168]: I would advise to have it, it should be 404 but there could be some bugs that may prevent to propagate the correct context back. @charlieR3 - do you know what would be the outcome if the key is not found or if it's misformatted (e.g. length us es than 12)

Comment (alexey-kadyrov-r3) [reply components/membership/membership-http-rpc/src/main/kotlin/net/corda/membership/httprpc/v1/KeysRpcOps.kt:168]: But for now that should be enough

**Review by alexey-kadyrov-r3 — DISMISSED** [review]
LGTM

**Pull request merged**